internal/middleware/pertoken/pertoken.go

```diff
@@ -19,7 +19,7 @@ import (
 
 const (
 	gcWindow             = 1 * time.Hour
-	revisionQuantization = 10 * time.Millisecond
+	revisionQuantization = 0 * time.Millisecond
```

Comment:
My concern with this change is that it would set a different default behaviour to `spicedb serve` and set wrong expectations. Why isn't asking folks to do `FullyConsistent` requests or use `AtLeastAsFresh` the recommended strategy in testing?

Reply:
I concur. This was intentional. Better to find any problems related to consistency in test than in production. If we were to try to solve it, I think the best way would be to have all timestamps requested before the "handover" point resolve to the handover snapshot, so at least the preloaded data is already consistent.

Reply:
> Why isn't asking folks to do FullyConsistent requests or use AtLeastAsFresh the recommended strategy in testing?

Because there is an expectation that preloaded data is already "present" in the system; you can't get a ZedToken for it, and there shouldn't be an expectation to need to make a FullyConsistent request for data that was already there if it has been there for (simulated) days or years

> I think the best way would be to have all timestamps requested before the "handover" point resolve to the handover snapshot, so at least the preloaded data is already consistent.

how would we do so?

Reply:
As discussed, squashed the memdb revisions